Samples/FNSCandyStyleTransfer/UWP/cs/MainPage.xaml.cs

```diff
@@ -110,6 +114,39 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 
             // Select first style
             UIStyleList.SelectedIndex = 0;
+
+            // Create a 1 second timer
+            _FramesPerSecondTimer.Tick += _FramesPerSecond_Tick;
+            _FramesPerSecondTimer.Interval = new TimeSpan(0, 0, 1);
+            _FramesPerSecondTimer.Start();
+        }
+
+        private void _FramesPerSecond_Tick(object sender, object e)
+        {
+            // how many seconds has it been?

```

Comment:
We calculate seconds in _FramesPerSecond_tick because even though this method is queued every second by DispatcherTimer, it doesn't mean that it will be called in exactly every second right?

Reply:
i added more comments.   you are correct !